transforms/components/index.js

```diff
@@ -12,12 +12,17 @@ const getPropertyName = ({ name: propertyName }) => {
   return name;
 };
 
-const formatTypeApplication = (applications, expression) => ({
-  type: expression.name,
-  items: {
-    type: applications[0].name,
-  },
-});
+const addTypeApplication = (applications, expression) => {
+  if (!applications && !expression) {
```

Comment:
Maybe we can add this. Is not very important.
```suggestion
  if (!applications && !expression) return {};
```